xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -203,9 +264,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Reads the next JSON token value from the source and parses it to a <see cref="T:System.Double" />.</summary>
+        <returns>The UTF-8 encoded token value parsed to a <see cref="T:System.Double" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `GetDouble` method throws an exception if the entire UTF-8 encoded token value cannnot be parsed to a <xref:System.Double> value.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">
+            The JSON token value isn't a <see cref="F:System.Text.Json.JsonTokenType.Number" />.</exception>
+        <exception cref="T:System.FormatException">
+            The JSON token value represents a number less than <see cref="F:System.Double.MinValue" /> or greater than <see cref="F:System.Double.MaxValue" />.</exception>
```

Comment:
This exception doesn't happen on .NET Core.  On Core it returns PositiveInfinity or NegativeInfinity (as appropriate).

Reply:
This came as-is from the source: https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs#L134

And the code seems to call ThrowHelper.GetFormatException no?

Reply:
It's fine to keep it as is for now. This might change pending https://github.com/dotnet/corefx/issues/33997, which is what @bartonjs is referring to.